handbook/engineering/code-intelligence/goals.md

```diff
@@ -22,120 +22,76 @@ Progress on adoption and usage is tracked in our [Looker dashboard](https://sour
 
-1. 🔄 Build lsif-clang testing infrastructure
-    - 🔄 Build a tool which can to semantically compare and diff LSIF dumps.
-    - 🔄 Write a test harness for lsif-clang which can perform snapshot tests on small code snippets.
-    - Spike: Explore how to perform snapshot tests on large projects for integration testing.
+1. ✅ Build lsif-clang testing infrastructure.
     
     **Outcome:** We can confidently make code changes to lsif-clang without worrying about obvious bugs.
 
-1. Sourcegraph Cloud has precise code intelligence for C++ repositories that use Bazel as a build system.
-    - Support Bazel projects in lsif-clang.
-    - Auto-index 5 repositories on Sourcegraph Cloud that use Bazel as a build system.
-    - Update and improve documentation to ensure a good user experience.
+1. 🔄 Sourcegraph Cloud has precise code intelligence for C++ repositories that use Bazel as a build system.
     
     **Outcome:** 5 popular open source C++ repositories using Bazel have up-to-date precise code intelligence on Sourcegraph Cloud.
     
 1. 🔄 Increase C++ precise code intel monthly operations.
-    - 🔄 Deliver to [*N<sub>1</sub>*][N1].
-    - Deliver to [*N<sub>2</sub>*][N2].
-    - Deliver to [*N<sub>3</sub>*][N3].
     
     **Outcome:** 3 enterprise customers with C++ precise code intel configured in their repositories.
 
 1. Sourcegraph Cloud has up-to-date precise code intelligence for C++ repositories that exceed 1mil SLOC.
-    - Create benchmarks of lsif-clang memory and CPU usage for various sizes of repositories. 
-    - Auto-index 5 repositories on Sourcegraph Cloud that exceed 1M SLOC.
     
     **Outcome:** 5 popular open source C++ repositories that exceed 1M SLOC have up-to-date precise code intelligence on Sourcegraph Cloud.
 
 1. Implement standard C++ code navigation features.
-    - Jump to declaration.
-    - Jump to implementation.
-    - Switch between source and header file.
 
 ### Java precise code intelligence support
 
 **Milestones:**
 
-1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Gradle repositories.
-    - ✅ Automatic inference for projects that use the Java plugin
-    - Automatic inference for projects that use the Android plugin
-    - Publish Gradle plugin.
-    - 🔄 Update and improve documentation to ensure a good user experience.
-    
+1. ✅ Sourcegraph Cloud has up-to-date precise code intelligence for 5 Gradle repositories.
+
     **Outcome:** 5 popular open source Gradle repositories have up-to-date precise code intelligence on Sourcegraph Cloud.
 
-1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Maven repositories.
-    - ✅ Automatic inference for projects that use the Java compiler plugin
-    - Automatic inference for projects that use the Scala/Kotlin plugins
-    - Publish Maven plugin.
-    - 🔄 Update and improve documentation to ensure a good user experience.
-    
+1. ✅ Sourcegraph Cloud has up-to-date precise code intelligence for 5 Maven repositories.
```

Comment:
```suggestion
1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Maven repositories.
```
Work for Maven support is complete but we still need to index repos on Cloud to mark milestone completion